swagger.yaml

```diff
@@ -1557,6 +1557,110 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/hosts/{host_id}/ignition:
+    get:
+      tags:
+        - installer
+      summary: Get the customized ignition file for this host
+      operationId: GetHostIgnition
+      parameters:
+        - in: path
+          name: cluster_id
+          type: string
+          format: uuid
+          required: true
+        - in: path
+          name: host_id
+          type: string
+          format: uuid
+          required: true
+      responses:
+        200:
+          description: Success.
+          schema:
+            $ref: '#/definitions/host-ignition-params'
+        401:
+          description: Unauthorized.
+          schema:
+            $ref: '#/definitions/infra_error'
+        403:
+          description: Forbidden.
+          schema:
+            $ref: '#/definitions/infra_error'
+        404:
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+        405:
+          description: Method Not Allowed.
+          schema:
+            $ref: '#/definitions/error'
+        409:
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+        500:
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+
+  /clusters/{cluster_id}/hosts/{host_id}/ignition/file:
+    get:
+      tags:
+        - installer
+      security:
+        - userAuth: []
```

Comment:
Do we ever expect a user to call this API?

Reply:
I doubt it given that we have the non-download endpoint as well, but it doesn't hurt to allow it.
Do you want me to also add user auth here?

Reply:
Oh, sorry misread the comment, I think I copied the permissions from the `/clusters/{cluster_id}/downloads/files` endpoint, but I guess that makes more sense because it includes the kubeconfig.

You think I should remove it here?

Reply:
Removed